customize_image.sh

```diff
@@ -82,7 +106,7 @@ sed -r -i -e 's/#hdmi_force_hotplug=1/hdmi_force_hotplug=1/' \
 	  $IMAGEDIR/boot/config.txt
 
 # limit disk space occupied by logs
-ln -s ../cron.daily/logrotate $IMAGEDIR/etc/cron.hourly
+[ -f ../$IMAGEDIR/etc/cron.hourly/logrotate ] || ln -s ../cron.daily/logrotate $IMAGEDIR/etc/cron.hourly
```

Comment:
not quite sure this should be checked.
as I know, with this check  the repeatly customize an image might be works, but the result will not work. 
This checking might delay the finding of the error.

Reply:
Without checking the build with raspios-full image will fail as this is file is already existing .

Reply:
Ok, I think we need a ticket for this. Why does the file exist? Is it the identical link or a different?